rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/internal/LRARecoveryModule.java

```diff
@@ -130,6 +137,41 @@ private void doRecoverTransaction(Uid recoverUid) {
         }
     }
 
+    private void moveEntryToFailedLRAPath (final Uid failedUid) throws ObjectStoreException {
+        String failedLRAType = LongRunningAction.getFailedLraType();
+        InputObjectState inputState = _recoveryStore.read_committed(failedUid, _transactionType);
+        InputObjectState failedLRAUidState = _recoveryStore.read_committed(failedUid, failedLRAType);
+        boolean result = false;
+
+        if (inputState != null) {
+            if (failedLRAUidState != null) {
+                // Record already exists in failedLRARecord_PATH, hence removing it from the LRA_PATH
+                boolean rmRec = _recoveryStore.remove_committed(failedUid, _transactionType);
+                if (!rmRec) {
+                    LRALogger.logger.warnf("Unable to remove the failed duclicate failed LRA record (Uid: %s) " +
+                            "(which is already present in the failed LRA record location type: %s.) from LRA_PATH: %s",
+                            failedUid, failedLRAType, _transactionType);
+                    return;
+                }
+                LRALogger.logger.infof("The failed lra record (Uid: %s) is already present in the failed LRA record location type: %s." +
+                        "Hence removed the failed duclicate record from LRA_PATH: %s", failedUid, failedLRAType, _transactionType);
+                return;
+            }
+
+            boolean moved = _recoveryStore.write_committed(failedUid, failedLRAType, new OutputObjectState(inputState));
+
+            if (!moved) {
+                LRALogger.logger.warnf("Unable to move failed lra record (Uid: %s) to new location type: %s", failedUid, failedLRAType);
```

Comment:
The same what I mentioned above - the `warn` messages should go to `lraI18NLogger`.

Reply:
ok